tests/modelbench_tests/test_run.py

```diff
@@ -162,11 +165,13 @@ def test_benchmark_multiple_suts_produces_json(self, runner, version, locale, pr
             version, locale if locale else Locale.EN_US, prompt_set if prompt_set else "practice", "default"
         )
 
+        FakeSUT
```

Comment:
What's this?

Reply:
Oops, that shouldn't be there. Thanks for catching it.